src/main/java/seedu/address/model/commons/WanderlustDate.java

```diff
@@ -55,7 +56,7 @@ public LocalDate getValue() {
 
     @Override
     public String toString() {
-        return value.toString();
+        return value.format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
```

Comment:
Is it supposed to be MM instead of 3 'M's?

Reply:
ohh its abbreviated form!

Reply:
yeap! It's meant to be Jan, Feb, etc.